core/templates/pages/exploration-player-page/layout-directives/feedback-popup.component.ts

```diff
@@ -83,7 +83,7 @@ export class FeedbackPopupComponent {
           this.feedbackSubmitted = true;
           setTimeout(() => {
             this.close();
-          }, 2000);
+          }, 3000);
```

Comment:
2 seconds was too low for the tooltip message that appears, so increasing it to 3 seconds, which would also give more time to fetch the message in e2e tests.